src/NumericalAlgorithms/Interpolation/InterpolationTarget.hpp

```diff
@@ -97,76 +103,59 @@ namespace intrp {
 ///      The dimension of the Domain.
 template <class Metavariables, typename InterpolationTargetTag>
 struct InterpolationTarget {
+  struct RegistrationHelper {
+    template <typename ParallelComponent, typename DbTagsList,
+              typename ArrayIndex>
+    static std::pair<observers::TypeOfObservation, observers::ObservationId>
+    register_info(const db::DataBox<DbTagsList>& /*box*/,
+                  const ArrayIndex& /*array_index*/) noexcept {
+      observers::ObservationId fake_initial_observation_id{
+          0.,
+          // Currently this ignores anything in
+          // post_interpolation_callback::observation_types except the first
+          // element.  This will be changed in an upcoming PR, which
+          // will modify RegisterSingletonWithObserverWriter.
+          tmpl::front<typename InterpolationTargetTag::
+                          post_interpolation_callback::observation_types>{}};
+      return {
+          observers::TypeOfObservation::Reduction,
+          std::move(fake_initial_observation_id)  // NOLINT
```

Comment:
why is the move and NOLINT here? If it's a trivially copyable type just copy it, or put a clang-tidy comment so we know what you're linting, please.